### PR TITLE
Issue #1078 Provide feedback about cache getting deleted

### PR DESCRIPTION
**Pull request by dharmit**
Addresses issue #1078

### CODE REVIEW

File: cmd/minishift/cmd/delete.go [hunk 1]
```diff
@@ -61,6 +60,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	if clearCache {
+		fmt.Println("Deleting Minishift cache")
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/delete.go]: I think we need to be more verbose than that e.g. We should say what files are getting deleted or atleast which directories.

**Comment by hferentschik** [comment]
add to whitelist

File: cmd/minishift/cmd/delete.go [hunk 2]
```diff
@@ -61,6 +60,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	if clearCache {
+		fmt.Printf("Deleting Minishift ISO, OpenShift container images and OpenShift origin binaries from %s/cache\n", constants.Minipath)
```
Comment (hferentschik) [inline cmd/minishift/cmd/delete.go]: hmm, whether you have images will depend really. I thought more about literally logging what gets deleted. The concrete file names. Not sure what @LalatenduMohanty had in mind.

Comment (LalatenduMohanty) [inline cmd/minishift/cmd/delete.go]: @dharmit @hferentschik I think we would be fine with the absolute path of the cache directory as mentioning individual files will make the output cluttered. I modified the code little bit (as below) and it looks fine to me.
```
if clearCache {
		cachePath := constants.MakeMiniPath("cache")
		err := os.RemoveAll(cachePath)
		if err != nil {
			atexit.ExitWithMessage(1, fmt.Sprintf("Error deleting Minishift cache: %v", err))
		} else {
			fmt.Printf("Removed the cache at: %s\n", cachePath)
		}
	}
```
I moved the print statement after deletion as I think saying deleting the cache would means we should again mention that it is deleted successfully.  Also as user has explicitly used --clear-cache we can just mention after deleting and no need to say we are deleting the cache. 
```
$ minishift delete --clear-cache
Removed the cache at: /home/lmohanty/.minishift/cache
Deleting the Minishift VM...
Minishift VM deleted.
```

Comment (gbraad) [inline cmd/minishift/cmd/delete.go]: The message should be concise (and simple): "Removed cache", or as suggested above "Removed cache at ~/.minishift/cache"

**Comment by hferentschik** [comment]
Merged, thanks @dharmit for another contribution. Keep it coming :-)

**Pull request closed without merging**